build.gradle.kts

```diff
@@ -407,6 +410,19 @@ project(":grazie") {
     }
 }
 
+project(":js") {
+    intellij {
+        version = ideaVersion
```

Comment:
Do we really need IDEA here? As I remember, JavaScript plugin is bundled into CLion as well

Reply:
Thanks, `baseVersion` should be used instead, right?

Reply:
Yep but you shouldn't specify it explicitly here since `baseVersion` is set as a default `version` [here](https://github.com/intellij-rust/intellij-rust/blob/953cd3d052383e4eb10ab048a10ad992cd91fc08/build.gradle.kts#L75)